rectools/visuals/visual_app.py

```diff
@@ -128,11 +162,15 @@ def __init__(
         auto_display: bool = True,
         formatters: tp.Optional[tp.Dict[str, tp.Callable]] = None,
         rows_limit: int = 20,
+        min_width: int = 100,
         **kwargs: tp.Any,
     ) -> None:
         self.rows_limit = rows_limit
         self.formatters = formatters if formatters is not None else {}
         self.data_storage: _AppDataStorage = self._create_data_storage(*args, **kwargs)
+        if min_width < 10:
```

Comment:
please, let's not hardcode numbers

At least let's put it into a constant

Reply:
fixed